templates/vue/src/components/Lightbox/media/ActivityMedia/AudioRecorder.vue

```diff
@@ -6,7 +6,11 @@
     Please provide microphone access to record your answer.
   </div>
   <div v-else class="recorder">
-    <audio v-if="state === states.DONE" controls :src="audio"></audio>
+    <audio
+      v-if="showAudio"
```

Comment:
It would be easier to read this if you place the condition directly here rather than in a computed value.
```suggestion
      v-if="state === states.DONE || state === states.WAITING_TO_BE_SENT"
```